pt-BR/mail/chrome/messenger/dataPane.dtd

```diff
@@ -1,77 +1,77 @@
+<!ENTITY dataPaneEvent.label          "Eventos">
 <!ENTITY dataPaneURL.label            "Links">
-<!ENTITY dataPaneAddresses.label      "Addresses">
+<!ENTITY dataPaneAddresses.label      "Endereços">
 
 <!-- data pane attachment popup -->
 <!-- XX Localisation note: Do not translate service names -->
-<!ENTITY dataPaneContext.uploadTo.label               "Upload to...">
+<!ENTITY dataPaneContext.uploadTo.label               "Enviar para...">
 <!ENTITY dataPaneContext.uploadToPicasa.label         "Picasa">
 <!ENTITY dataPaneContext.uploadToPicasa.accesskey     "P">
 <!ENTITY dataPaneContext.uploadToPhotobucket.label    "Photobucket">
 <!ENTITY dataPaneContext.uploadToPhotobucket.accesskey "b">
 <!ENTITY dataPaneContext.uploadToGoogleDocs.label     "Google Docs">
 <!ENTITY dataPaneContext.uploadToGoogleDocs.accesskey "G">
-<!ENTITY dataPaneContext.openAttachment.label         "Open">
+<!ENTITY dataPaneContext.openAttachment.label         "Abrir">
 <!ENTITY dataPaneContext.openAttachment.accesskey     "O">
-<!ENTITY dataPaneContext.saveAsAttachment.label       "Save As...">
+<!ENTITY dataPaneContext.saveAsAttachment.label       "Salvar como...">
 <!ENTITY dataPaneContext.saveAsAttachment.accesskey   "a">
 <!ENTITY dataPaneContext.quickLookAttachmentMac.label     "Quick Look...">
 <!ENTITY dataPaneContext.quickLookAttachmentMac.accesskey "Q">
-<!ENTITY dataPaneContext.quickLookAttachment.label    "Preview...">
+<!ENTITY dataPaneContext.quickLookAttachment.label    "Visualizar...">
 <!ENTITY dataPaneContext.quickLookAttachment.accesskey "r">
-<!ENTITY dataPaneContext.detachAttachment.label       "Detach...">
-<!ENTITY dataPaneContext.deleteAttachment.label       "Delete">
-<!ENTITY dataPaneContext.saveAll.label                "Save All...">
+<!ENTITY dataPaneContext.detachAttachment.label       "Retirar...">
+<!ENTITY dataPaneContext.deleteAttachment.label       "Excluir">
+<!ENTITY dataPaneContext.saveAll.label                "Salvar todos...">
 <!ENTITY dataPaneContext.saveAll.accesskey           "v">
-<!ENTITY dataPaneContext.systemPhotoImportMac.label  "Add to iPhoto">
+<!ENTITY dataPaneContext.systemPhotoImportMac.label  "Adicionar ao iPhoto">
 <!ENTITY dataPaneContext.systemPhotoImportMac.accesskey  "i">
 
 <!-- contact panel -->
-<!ENTITY contactPanel-name.label  "name:">
-<!ENTITY contactPanel-title.label "title:">
-<!ENTITY contactPanel-company.label  "company:">
+<!ENTITY contactPanel-name.label  "nome:">
+<!ENTITY contactPanel-title.label "título:">
+<!ENTITY contactPanel-company.label  "empresa:">
 
-<!ENTITY contactPanel-phone.label "phone:">
-<!ENTITY contactPanel-work.label "(work)">
-<!ENTITY contactPanel-home.label "(home)">
-<!ENTITY contactPanel-mobile.label "(mobile)">
+<!ENTITY contactPanel-phone.label "telefone:">
+<!ENTITY contactPanel-work.label "(empresa)">
+<!ENTITY contactPanel-home.label "(residencial)">
+<!ENTITY contactPanel-mobile.label "(celular)">
 
 <!ENTITY contactPanel-email.label "email:">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY contactPanel-email.label "correio eletrônico:">